OWNERS

```diff
@@ -1,28 +1,8 @@
 approvers:
   - abhi-g
-  - aronchick
-  - dansanche
-  - ewilderj
-  - jinchihe
   - jlewi
-  - kunmingg
-  - lluunn
-  - nickchase
-  - pdmack
   - richardsliu
   - sarahmaddox
-  - texasmichelle
-  - willingc
-  - dsdinter
-  - Ark-kun
-  - gaoning777
-  - hongye-sun
-  - IronPan
-  - neuromage 
-  - paveldournov
-  - qimingj
-  - rileyjbauer
-  - vicaire
   - joeliedtke
```

Comment:
Not sure of the role for joeliedtke?

Reply:
Maybe part of Tech writers team?

Reply:
@animeshsingh That's right, Joe is the tech writer for Kubeflow Pipelines.